examples/splunk-hec/logging/main.go

```diff
@@ -0,0 +1,36 @@
+package main
+
+import (
+	"os"
+	"os/signal"
+	"syscall"
+	"time"
+
+	"go.uber.org/zap"
+)
+
+
+func main() {
+	logger, _ := zap.NewProduction()
+	defer logger.Sync()
+
+	counter := int64(0)
+	c := make(chan os.Signal, 1)
+	signal.Notify(c, os.Interrupt, syscall.SIGTERM, syscall.SIGQUIT)
+	ticker := time.NewTicker(1 * time.Second)
+
+	logger.Info("Start logging app")
+	for {
+		select {
+		case <-ticker.C:
+			time.Sleep(1 * time.Second)
```

Comment:
Is this still desired when triggered by the ticker?

Reply:
No, you're right, I can remove that line.

Reply:
Done.